frontend/src/features/contacts/list/ContactSearchResults/columns.tsx

```diff
@@ -104,16 +104,26 @@ const columns: ColumnWithProps<IContactSearchResult>[] = [
     accessor: 'controls' as any, // this column is not part of the data model
     width: 40,
     maxWidth: 40,
-    Cell: (props: CellProps<IContactSearchResult>) => (
-      <StyledDiv>
-        <IconButton variant="light">
-          <MdEdit size={22} />
-        </IconButton>
-        <IconButton variant="light">
-          <MdContactMail size={22} />
-        </IconButton>
-      </StyledDiv>
-    ),
+    Cell: (props: CellProps<IContactSearchResult>) => {
+      const history = useHistory();
+
+      return (
+        <StyledDiv>
+          <IconButton
```

Comment:
Do these need to be controlled by claims?

Reply:
They weren't originally but can change. All I did was add an **onClick** handler to existing button as my story spoke about clicking on the button. I didn't go to the contact list story to check claims but could see that they are required here.

They should have been hidden since then but can change now